float32 height
string[] objects
string label